.changeset/calm-tables-hide.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `selector-pseudo-class-allowed-list` end positions and message ([#6262](https://github.com/stylelint/stylelint/pull/6262))
```

Comment:
```suggestion
Fixed: `selector-pseudo-class-allowed-list` end positions and message
```

Should these messages not include the pull request number as that (and the mention) is automatically added by the script?

Reply:
In this PR, it can't, because the changesets are generated in this PR. 🤣

Reply:
> In this PR, it can't, because the changesets are generated in this PR. 🤣

Oh, I see. Makes sense.

> Do you want me to add the mentions manually in this PR?

Narh, it's not worth it if it's manual.  They can start appearing next time around when it's automated.